docs/docs/user-guide/run-cli-in-docker-container.md

```diff
@@ -0,0 +1,101 @@
+# Run CLI for Microsoft 365 in a Docker Container
+
+You can use Docker to run a standalone Linux container with CLI for Microsoft 365 and PowerShell pre-installed, with command completion (tab) automatically configured for you in both bash and PowerShell, without having to install any of the required dependencies on your host machine.
+
+## Prerequisites
+
+To use the published Docker images, you will first need to have Docker installed and configured on your host machine. Please refer to the Docker documentation on how to [install Docker](https://docs.docker.com/get-docker/) on Windows, Mac and Linux.
+
+## Install and run latest
+
+To install and run the latest version of CLI for Microsoft 365, use the `docker run` command and specify the CLI docker image name.
+
+```
+docker run --rm -it m365pnp/cli-microsoft365:latest
+```
+
+Executing this command for the first time will download the image onto your machine and start the container and invoke an interactive session automatically `(-it)`, displaying a bash shell prompt. 
+
+You can exit from the prompt by closing the terminal window or typing `exit`. When you exit the container will be automatically stopped to free up system resources `(--rm)`.
```

Comment:
```suggestion
You can exit from the prompt by closing the terminal window or typing `exit`. When you exit the container, it will be automatically removed to free up system resources `(--rm)`.
```